core-java-modules/core-java-arrays-convert/src/test/java/com/baeldung/array/conversions/ByteArrayToPrimitiveByteArrayUnitTest.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.array.conversions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.commons.lang3.ArrayUtils;
+import org.junit.jupiter.api.Test;
+
+public class ByteArrayToPrimitiveByteArrayUnitTest {
+    private static final byte[] EXPECTED_ARRAY_VALUES = {65, 66, 67, 68};
+    private static final Byte[] BYTE_ARRAY = {65, 66, 67, 68};
+
+    @Test
+    public void givenByteArray_whenConvertingUsingByteValue_thenGiveExpectedResult() {
+        byte[] newByteArray = new byte[BYTE_ARRAY.length];
+        for (int i = 0; i < BYTE_ARRAY.length; i++) {
+            newByteArray[i] = BYTE_ARRAY[i].byteValue();
+        }
+        assertThat(newByteArray)
```

Comment:
you can compress the whole assertion in one line:

_assertThat(newByteArray).containsExactly(EXPECTED_ARRAY_VALUES);_

please, apply to all assertions in both classes

Reply:
Fixed